src/local/index.coffee

```diff
@@ -12,9 +12,10 @@ Watcher = require './watcher'
 # It also applied changes from the remote cozy on the local filesystem.
 class Local
     constructor: (config, @prep, @pouch) ->
+        deviceName = config.getDefaultDeviceName()
         @basePath = config.getDevice().path
         @tmpPath  = path.join @basePath, ".cozy-desktop"
-        @watcher  = new Watcher @basePath, @prep, @pouch
+        @watcher  = new Watcher @basePath, @prep, @pouch, deviceName
```

Comment:
I prefer:

```coffee
@watcher  = new Watcher @basePath, @prep, @pouch
```

The Watcher from this file is the local watcher and doesn't use at all the deviceName.